apps/site/package.json

```diff
@@ -3,9 +3,10 @@
   "type": "module",
   "scripts": {
     "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "dev": "cross-env NODE_NO_WARNINGS=1 next dev",
+    "dev": "npm run build-blog-data:watch & sleep 1 && cross-env NODE_NO_WARNINGS=1 next dev",
```

Comment:
If I run this, and then exit the command, will it still be running in the background?

Reply:
no, killing the command kills everything, I tried that this on macos and it definitely kills the watch process as well

I _think_ this also applies to any other OS 🤔